drivers/wifi/nrf700x/zephyr/src/zephyr_wifi_mgmt.c

```diff
@@ -553,6 +553,10 @@ int wifi_nrf_set_twt(const struct device *dev,
 
 	switch (twt_params->operation) {
 	case WIFI_TWT_SETUP:
+		if (vif_ctx_zep->twt_in_progress) {
+			return WIFI_TWT_FAIL_OPERATION_IN_PROGRESS;
```

Comment:
Where is this defined? I guess you need a downstream commit for https://github.com/zephyrproject-rtos/zephyr/pull/55581/files

Reply:
Yes, I will do that before giving merge request for this one